### PR TITLE
[DE] Translate Resources

### PR DESCRIPTION
**Pull request by DerSchlupf**
New german translation page of ressource section:
image

### CODE REVIEW

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
image

**Comment by qwrtln** [comment]
If you compare the first paragraph with the English original, you can see that the translation is significantly longer. You should shorten it. It doesn't do much harm here, it just shrinks the Griffin image, but in other places it will break the layout.

File: translations/resources.tex/de.po [hunk 1]
```diff
@@ -75,6 +75,23 @@ msgid ""
 "  \\includegraphics[width=0.8\\linewidth]{\\images/valuables.png}\n"
 "  \\footnotesize\\textcolor{darkcandyapplered}{\\textit{\\textbf{Valuables \\phantom{Materials}}}}"
 msgstr ""
+"\\pagetarget{Resources}{Es gibt}\\index{Ressourcen} drei Arten von Ressourcen im Spiel: Gold \\svg{gold}, Baumaterialien \\svg{building_materials}, und Minerale \\svg{valuablegreater}.\n"
+"Ressourcen werden während des Spiels ausgegeben, um deine \\pagelink{Town}{Stadt}zu erweitern, um \\pagelink{Units}{Einheiten} zu rekrutieren, und um \\pagelink{spells}{Zaubersprüche} zu kaufen.\n"
```
Comment (qwrtln) [inline translations/resources.tex/de.po:79]:
```suggestion
"Ressourcen werden während des Spiels ausgegeben, um deine \\pagelink{Town}{Stadt} zu erweitern, um \\pagelink{Units}{Einheiten} zu rekrutieren, und um \\pagelink{spells}{Zaubersprüche} zu kaufen.\n"
```

File: translations/resources.tex/de.po [hunk 2]
```diff
@@ -104,22 +126,22 @@ msgstr ""
...
 #: sections/resources.tex:51
 msgid "[{\\svg[15]{2_building_materials}}] — 2 × Building Materials"
-msgstr ""
+msgstr "[{\\svg[15]{2_building_materials}}] — 2 × Baumaterialien"
 
 #. type: itemize
 #: sections/resources.tex:51
 msgid "[{\\svg[15]{4_building_materials}}] — 4 × Building Materials"
-msgstr ""
+msgstr "[{\\svg[15]{4_building_materials}}] — 4 × Building Materials"
```
Comment (qwrtln) [inline translations/resources.tex/de.po:134]:
```suggestion
msgstr "[{\\svg[15]{4_building_materials}}] — 4 × Baumaterialien"
```

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by qwrtln** [comment]
@ElDany29 Please review this.

**Comment by DerSchlupf** [comment]
Updatet page see above. I have correct tiping errors. First paragrapg isn't really shorter than before. Its difficult to do it without making the rules more unclear.

**Comment by DerSchlupf** [comment]
po4a commit

**Comment by DerSchlupf** [comment]
Picture updatet!

File: translations/resources.tex/de.po [hunk 3]
```diff
@@ -83,18 +101,23 @@ msgid ""
...
+msgstr "\\textbf{\\textit{Einkommensleisten}}"
 
 #. type: Plain text
 #: sections/resources.tex:51
 msgid ""
 "Possible Resource Die\\index{Resource Die}\\index{Dice} \\svg{resource_die} "
 "results:"
 msgstr ""
+"Ergebnisse des Ressourcenwürfels\\index{Ressourcenwürfel}\\index{Würfel} "
+"\\svg{resource_die}"
```
Comment (qwrtln) [inline translations/resources.tex/de.po:120]:
```suggestion
"\\svg{resource_die}:"
```

**Pull request merged**